simulator/programming/index.md

```diff
@@ -27,61 +27,72 @@ Your code should be developed in `robot.py`.
   Only your controller code will be present in the competition environment.
 </div>
 
-## Programming Interface
+## Robot
 
-Unless otherwise stated, the simulator’s API is the same as the real SR API described in the [programming docs]({{ site.baseurl }}/programming/).
+There is a pre-built robot used in the simulator.
+To allow this simulated robot to move around and sense its environment a set of motors and sensors have been connected as detailed below.
+
+The simulator’s API is very similar to the real SR API described in the [programming docs]({{ site.baseurl }}/programming/), with the exception of `R.time` and `R.sleep`.
 
 ### Motors
 
-Your robot has one motor board attached, the left wheel in port `m0`, and the right wheel in `m1`.
+Your robot has one motor board attached, the left wheel is connected to the first port, and the right wheel to the second.
+
+The motor board has the part code `srABC1`, since only a single motor board is attached it can be referenced as `R.motor_board`.
+
+### Servos
+
+Your robot has one servo board attached, the jaws of the robot are controlled by a pair of servos:
+
+| Servo | Location  |
+|-------|-----------|
+| 0     | Left Jaw  |
+| 1     | Right Jaw |
+
+Setting each servo to -1 fully opens the respective jaw.
 
-The motor board does not have a part code, so it needs to be indexed using `0`.
+The servo board has the part code `srXYZ2`, since only a single servo board is attached it can be referenced as `R.servo_board`.
```

Comment:
```suggestion
The servo board has the part code `srXYZ2`, but since only a single servo board is attached it can be referenced as `R.servo_board`.
```